### PR TITLE
Complete and update the profile.

### PR DESCRIPTION
**Pull request by tresoldi**
This PR fixes the only unmapped grapheme (k:) and formats the profile, removing some redundancy, sorting it, and removing a couple of trailing tabs in the original.

### CODE REVIEW

File: etc/orthography.tsv [hunk 1]
```diff
@@ -1,104 +1,97 @@
...
+ɖɖ	ɖː	9	U+0256 U+0256	aɖɖə,doɖɖadə,eɖɖedana
+ɖʰ	ɖʰ	1	U+0256 U+02B0	peɖʰ
+ə:	əː	3	U+0259 U+003A	kə:ɭ,pə:ɭ,ə:ɭ
+ɭɭ	ɭː	7	U+026D U+026D	ni:ɭɭu,oɭɭijadu,ʋeɭɭe
+ʈʈ	ʈː	45	U+0288 U+0288	muʈʈa,muʈʈi,paʈʈi
+ʋʋ	ʋː	5	U+028B U+028B	iʋʋaɖam,koʋʋu,oʋʋoɖimo
+ṭṭ	ʈː	6	U+1E6D U+1E6D	hoṭṭe,paṭṭa,vaṭṭa-
+kkʰ	kʰː	1	U+006B U+006B U+02B0	kʰekkʰa
+tʃ:	tʃː	1	U+0074 U+0283 U+003A	tʃ:ɭa
+dʒdʒ	dːʒ	3	U+0064 U+0292 U+0064 U+0292	bedʒdʒ,idʒdʒna,nedʒdʒa
```
Comment (LinguList) [inline etc/orthography.tsv]: This is not a bipa symbol. Bipa is `dʒː`.

File: etc/orthography.tsv [hunk 2]
```diff
@@ -1,104 +1,97 @@
...
+tʃtʃ	tʃː	33	U+0074 U+0283 U+0074 U+0283	katʃtʃa:,natʃtʃatiɾa,patʃtʃad
```
Comment (LinguList) [inline etc/orthography.tsv:93]: Here it is correct.

File: etc/orthography.tsv [hunk 3]
```diff
@@ -1,104 +1,97 @@
...
+bo:dʰ	b oː dʱ	1	U+0062 U+006F U+003A U+0064 U+02B0	bo:dʰ
+piva:ʰd	p i v aː dʱ	1	U+0070 U+0069 U+0076 U+0061 U+003A U+02B0 U+0064	piva:ʰd
+"vidʒdʒa"""	v i dːʒ a	1	U+0076 U+0069 U+0064 U+0292 U+0064 U+0292 U+0061 U+0022	"vidʒdʒa""
```
Comment (LinguList) [inline etc/orthography.tsv]: dito, see above my comment.

**Review by LinguList — COMMENTED** [review]
see my inline comments. They are easy to handle.

File: etc/orthography.tsv (hunk 1 above)
Comment (tresoldi) [reply etc/orthography.tsv]: I much prefer `dʒː`, but we have `dːʒ` in BIPA: https://clts.clld.org/parameters/long_voiced_post-alveolar_sibilant_affricate_consonant

File: etc/orthography.tsv (hunk 2 above)
Comment (tresoldi) [reply etc/orthography.tsv:93]: Yes, it is BIPA issue... The `dːʒ` was actually introduced by the normalization script, which replaces aliases with their grapheme.

**Review by SimonGreenhill — APPROVED** [review]
Great, thanks @tresoldi !

File: etc/orthography.tsv (hunk 1 above)
Comment (LinguList) [reply etc/orthography.tsv]: then this is an obvious error in BIPA. Where were you when we discussed this with @cormacanderson, @Tresoldi? It was a deliberate decision to switch from `dːʒ` to `dʒː`. So this slipped any review by then (and is obviously on me, but also shows why it is important that more people check CLTS for linguistic etc. consistency).

Can you please file an issue and make sure this is quickly changed in clts?

Comment (tresoldi) [reply etc/orthography.tsv]: Ok, using the indended BIPA with 99fc465 and opened an issue in clts-data.

**Pull request merged**